app/views/task/details.html.erb

```diff
@@ -5,19 +5,22 @@
     <tr>
       <th>Name</th>
       <th>Description</th>
+      <th>Assigned To</th>
       <th>Completed</th>
       <th>Date Completed</th>
+      <th></th>
     </tr>
   </thead>
   <tbody>
       <tr>
         <td><%= @task.name %></td>
         <td><%= @task.description %>
+        <td><%= @task.person_id %></td>
         <td><%= @task.completed %></td>
         <td>
           <%= @task.completed_at%>
         </td>
-      <td>
+      <td><a href="<%="/task_#{@task.id}/edit"%>">edit</a></td>
```

Comment:
Rails provides path helper methods for most of the routes that you create (which you may or may not have covered in class yet). So the what I usually see in this kind of code would look more like this:

```ruby
<td><a href="<%= edit_task_path(@task)%>">edit</a></td>
```

However, in order to get these helpers you need to set up your routes to match the convention that Rails uses so you don't have this at the moment. I wouldn't worry about it for now (although I do discuss routing a lot more below in the routes file). 

If you are interested, try running `rake routes` from your terminal (in the app folder). On the left you'll see the prefix that Rails gives you for the helpers, but it only sets the ones up that it can figure out based on it's convention.